packages/design-system/src/styles/components/_Table.scss

```diff
@@ -30,25 +30,31 @@ $table-stacked-header-max-width: 160px !default;
   th {
     border: 0;
     border-top: 1px solid;
+    /* stylelint-disable-next-line value-no-vendor-prefix */
+    display: -ms-grid;
```

Comment:
Hm, I'm wondering why our `browserlist` and `autoprefixer` setup doesnt do this automatically?

Reply:
I'm not sure yet. But even with the prefix, still not working correctly on ie 11 yet. Still trying to figure out why.

Reply:
IE11 only has partial support of grid https://caniuse.com/css-grid

Reply:
I found this resource too: https://dev.to/eblairmckee/css-grid-in-ie11-it-s-possible-and-not-as-hard-as-you-think-okp